docker/detect_rRNA/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda3
+
+LABEL Maintainer="Ekaterina Sakharova <kates@ebi.ac.uk>"
```

Comment:
I would use "Microbiome Informatics Team www.ebi.ac.uk/metagenomics". So we don't tie containers or code to individuals

Reply:
agree